geoviews/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@holoviz/geoviews",
-  "version": "1.12.0",
+  "version": "1.13.0-rc.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@holoviz/geoviews",
-      "version": "1.12.0",
+      "version": "1.13.0-rc.1",
       "license": "BSD-3-Clause",
       "dependencies": {
         "@bokeh/bokehjs": "~3.4.0",
```

Comment:
This should be 3.5

Reply:
Have opened here: https://github.com/holoviz/geoviews/pull/740

Reply:
Thanks; how did you get the other versions?